README.MD

```diff
@@ -106,7 +108,7 @@ b0934102c7a6        auction_center
 Now, let's connect to registry container via:
 
 ```bash
-$ docker exec -ti <CONTAINER ID> /bin/bash
+$ docker-compose run registry bash
```

Comment:
Doesn't 'docker-compose run registry bash' spin up another image, whereas 'docker exec  -ti <CONTAINER ID> /bin/bash' opens bash in already running container. IMHO the docker exec adds less load to ones computer.

Reply:
'docker exec -ti /bin/bash' if run container with this option, then container will always restart during setup. Therefore, ```docker-compose run registry bash ``` it's temporary solution for this issue.

Reply:
You're correct, exec uses existing container and run initializes a new one. My guess is that registry container went into restart loop for him because of some misconfiguration, be it missing certificaters, unsupplied config keys etc. Otherwise it would throw you right off from the terminal if it went into restart again. Maybe @OlegPhenomenon can ellaborate?